src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js

```diff
@@ -25,7 +25,11 @@ define(['css!suluautomationcss/main'], function() {
 
             'use strict';
 
+            config.set('sulu_automation', true);
```

Comment:
Shouldn't we use something like `sulu_automation.enabled` or something like this? This way it is hard to add more config options later.